### PR TITLE
Create PrimeNumber.c with time complexity and user input

### PR DESCRIPTION
**Pull request by manavbhikadiya**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by manavbhikadiya** [comment]
Changes complete with user input and time complexity. please accept my pull request.

File: C/math/PrimeNumber.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+int main() {
+   int num, flag = 0;
+   
+   printf("Enter number to check prime or not : ");
+   scanf("%d",&num);
+   
+   for(int i=2 ; i < num/2 ; i++) {
```
Comment (atarax665) [inline C/math/PrimeNumber.c:8]: We need to check for factors of the number only till ```sqrt(n)```. Refer [this](https://stackoverflow.com/questions/5811151/why-do-we-check-up-to-the-square-root-of-a-prime-number-to-determine-if-it-is-pr) to learn more.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Iterative approach will be fine after the suggested change is made. You also need to implement this recursively.

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**